doc/source/how-to-run-flower-on-azure.rst

```diff
@@ -0,0 +1,197 @@
+VM in the Microsoft Azure portal. There, we will click the ``Add inbound port rule``. In
+the new window that pops up we perform the following steps:
+
+- **Source**: we select ``IP Addresses``
+- **Source IP addresses/CIDR ranges**: we add the two Public IPs of our client VMs,
+  separated by comma
+- **Destination**: ``Any``
+- **Service**: ``custom``
+- **Destination port ranges**: ``9092``
+- **Protocol**: ``TCP``
+
+The rest of the fields can be left at their default values.
+
+Finally, we need to also open port 9093 to allow receiving and executing incoming
+application requests. To enable this we just need to repeat the steps above, i.e.,
+create a new inbound rule, where for port range we assign port 9093. If we already know
+the Public IP from which our local machine (e.g., laptop) will be submitting
+applications to the Azure cluster, then we just need to specify the Source IP
+address/CIDR range. However, if we want to keep the port widely open we simply need to
+change source to ``Any``.
+
+To be more precise, if we know the Public IP of our machine, then we need to change the
+following:
+
+- **Source IP addresses/CIDR ranges**: add machine's Public IP
+- **Destination port ranges**: ``9093``
+
+Otherwise, we just need to change the following:
+
+- **Source**: ``Any``
+- **Destination port ranges**: ``9093``
+
+Flower Environment
+------------------
+
+Configuration
+~~~~~~~~~~~~~
+
+Assuming we have been able to login to each VM, we need to make sure that the Flower
+environment is installed. To accomplish this, we first install Python in each machine
+and then create a path, referred to as ``<INSTANCE_PATH>`` in the code snippet below,
+which we will use to create inside a virtual environment, (e.g.,
+``<INSTANCE_PATH>/flower_venv-3.11``) and the Flower application project.
+
+To create the Flower application project, we need to run ``flwr run`` and then give a
+name to the project, e.g., ``flwr_azure_test``, give the name of the author and finally
+select the type of the Flower Framework we want to run, e.g., ``numpy``.
+
+.. code-block:: bash
+
+    sudo add-apt-repository ppa:deadsnakes/ppa
+    sudo apt update
+    sudo apt install python3.11
+    sudo apt install python3.11-venv
+    python3.11 -m venv <INSTANCE_PATH>/flower-venv-3.11
+    source activate <INSTANCE_PATH>/flower-venv-3.11/bin/activate
+    pip install flwr
+    cd <INSTANCE_PATH>
+    flwr new  # <-- give project name, user name, and select framework type
+    # Edit the files under the ``flwr_azure_test`` project, e.g., define server and client logic.
+
+Server Initialization
+~~~~~~~~~~~~~~~~~~~~~
+
+After configuring the Flower environment, we can proceed by starting the long-running
+processes at each VM instance. In particular we need to run the following commands,
+first in the server (``SuperLink``) and then at each client (``SuperNode``); assuming
+the virtual environment is enabled and we are inside the <INSTANCE_PATH> we created
+earlier.
+
```

Comment:
Before this code block i'd include a `.. note::` that tells readers to check the bottom of the page for links on how to run the same but with TLS and auth. Or even better, include those links in the note message.

Reply:
I think your suggestion to add them as notes altogether is better. I will remove them from the Additional Resources section.